editor/import_dock.cpp

```diff
@@ -572,6 +572,12 @@ void ImportDock::_reimport() {
 		Error err = config->load(params->paths[i] + ".import");
 		ERR_CONTINUE(err != OK);
 
+		// Added this to check if we have proper importer support for
+		// these files before we actually attempt to import it
+		// This will allow user to set blender_path and fbx2gltf_paths
+		// if they reimport those files and don't have the settings set
+		EditorFileSystem::get_singleton()->scan_import_support(params->paths);
```

Comment:
I'm a bit wary of changes to the core reimport rountine. I feel this might be easier to review as a separate PR, keeping only the direct Blender changes in this PR.

Reply:
I think if I don't have this code, The editor does not work as intended, I don't remember what the use case was, but it was something to do with blender and fbx import not working when those assets were in the project.

I can move this into a different PR for now though as it does not affect the main goal of the PR.